### ISSUE TITLE
Possible work around/Eshop access on region changed 3DS

### ISSUE THREAD

**Issue by akirayuki** [Plailect/Guide#798]
I managed to get eshop access/home-screen game updates and even check system transfer(although I haven't done any transfer yet since I need 2 more days for another transfer) on a region changed N3DS that previously connected to eshop.

My N3DS XL is originally JPN, it was second hand and has a couple of games installed on the home screen. I a9lh it and region changed it afterwards to EUR but as everyone else, I can't access eshop/update the games via homescreen because of the error code im getting(007-2001). The method I used for region changing was the old one, where you have to install cias of the specific region you want it to be.

Just recently, I bought a JPN white n3ds on 11.0, so what I did was restore my CFW N3DS XL to it's original JPN nand, system transfer it to the 11.0 JPN white n3ds and used dsiwarehax to a9lh it. It went smoothly and now both my JPN 3DS is already on a9lh. I also region changed the white n3ds to us but I didn't manage to connect to eshop though(even though I haven't connected it to eshop since it's brand new) but it's not a big issue for me.

I restored my EUR nand back up to my JPN N3DS XL so I can be able to play my games again. But out of curiosity, I tried opening Eshop since the "green circle" notification has been bugging me for months, but what do you know, It went through(which previously does not because of the error). I tried downloading Swap Doodle but it seems I need NNID for that so I made a new NNID, and I managed to download Swap Doodle afterwards. I also tried updating a game in the homescreen and it updated(I can't do this previously so that's the reason I had CIAngel back when there's no Freeshop yet).

So TLDR, it seems that N3DS that connected to eshop before region changing can access eshop after region changing. I'm not pretty sure about this, but if you need my NAND so you can check, I'll gladly send it to you.

Edit:
I managed to went through the part where the system transfer is looking for a target system now, and it's only connected to Local connection. Does it mean it passed Nintendo's check and considered it as a legit EUR N3DS now?

UPDATE:
Currently performing a System Transfer and it seems to be working fine!! Im gonna post the result once it's finished.

UPDATE2:
Confirmed, system transfer works using a region changed 3DS. Currently doing dsiwarehax now on a legit EUR N3DS.

**Comment by cubecubec** [comment]
i have a same issue as yours,N3DS XL JPN region changes to USA,and cannot access eshop,also my OLD 3ds XL JPN to USA region change had issue error code 007-2001.i hope you can find a way pleaseeeee really appreciated..

**Comment by gebgeb1** [comment]
Wow, thats awesome! so how exactly did it work I wonder? I would be really interested in getting my region changed JPN N3ds  to Connect to EShop

**Comment by akirayuki** [comment]
Im not really sure. I opened an issue about this so someone who has the right knowledge can look at what happened.

**Comment by Kanuan** [comment]
Before you did your first region change had you acessed the eshop?
 It is believed that only a N3DS that has never acessed the eshop can log into the eshop after a region change. So if you never did acess the eshop and got the error after your first region change than it was likely caused by other reasons.

**Comment by Plailect** [comment]
Lacking any insight into how this happened, I'll chalk it up to some kind of unknown server factor and mention it in the guide.

**Comment by helmergoza** [comment]
estimated friend

i have the same problem with 50 3ds japan to USA, did you get any solution to go to eshop after the change.??
